core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -391,8 +393,17 @@ public static Wallet fromWatchingKey(NetworkParameters params, DeterministicKey
     /**
      * Creates a wallet that tracks payments to and from the HD key hierarchy rooted by the given watching key. The
      * account path is specified. The key is specified in base58 notation and the creation time of the key. If you don't
-     * know the creation time, you can pass {@link DeterministicHierarchy#BIP32_STANDARDISATION_TIME_SECS}.
+     * know the creation time, you can pass empty and it will use {@link DeterministicHierarchy#BIP32_STANDARDISATION_TIME_SECS}.
```

Comment:
We should deprecate `DeterministicHierarchy#BIP32_STANDARDISATION_TIME_SECS` and define `public static final Instant DeterministicHierarchy#BIP32_STANDARDISATION_TIME`.

Reply:
We should also consider defaulting creationTime of Deterministic Keys to DeterministicHierarchy#BIP32_STANDARDISATION_TIME.

Reply:
> We should deprecate `DeterministicHierarchy#BIP32_STANDARDISATION_TIME_SECS` and define `public static final Instant DeterministicHierarchy#BIP32_STANDARDISATION_TIME`.

I agree. I saved this for a future commit.

Reply:
> We should also consider defaulting creationTime of Deterministic Keys to DeterministicHierarchy#BIP32_STANDARDISATION_TIME.

Yes, and we also could default single keys to the invention to Genesis time.

However, these changes are out of scope for this PR.

Reply:
> However, these changes are out of scope for this PR.

I agree, but they might help reduce some of the method-overload boilerplate.

Reply:
Honestly if we want to insert defaults we need to plan and think through carefully. And if you think it's a pre-requisite to this PR, we should talk about defaults first. I'll open a separate issue.

Reply:
Done: https://github.com/bitcoinj/bitcoinj/issues/2755